tests/util/sanitycheck/sanity_check.go

```diff
@@ -83,6 +88,36 @@ func setupTrafficTest(t framework.TestContext, revision string, ambient bool) (n
 	return testNs, client, server
 }
 
+func BlockTestWithPolicy(t framework.TestContext, client, server echo.Instance) {
+	ns := server.Config().Namespace.Name()
+	_, err := server.Config().Cluster.Istio().SecurityV1().AuthorizationPolicies(ns).Create(
```

Comment:
nit: normally we would do something like `t.ConfigIstio().Eval(...).Apply()`. Not a huge deal though

Reply:
Although this won't cleanup which may break things later? So maybe it is a big deal